### PR TITLE
feat: dashboards support

### PR DESCRIPTION
**Pull request by ecrupper**
Leverages SDK to manage dashboards. Add, update, view, and get all supported. Get will return the user's dashboards.

Was hoping to include the ability to add dashboards to a user, but the User resource is not supported in the SDK nor the CLI.

### CODE REVIEW

File: cmd/vela-cli/get.go [hunk 1]
```diff
@@ -32,6 +33,8 @@ var getCmds = &cli.Command{
 		// https://pkg.go.dev/github.com/go-vela/cli/command/build?tab=doc#CommandGet
 		build.CommandGet,
 
+		dashboard.CommandGet,
```
Comment (plyr4) [inline cmd/vela-cli/get.go:36]: omg no comment??? (im joking)

File: command/dashboard/view.go [hunk 2]
```diff
@@ -0,0 +1,92 @@
...
+
+// CommandView defines the command for viewing a dashboard.
+var CommandView = &cli.Command{
+	Name:        "dashboard",
+	Description: "Use this command to view a dashboard.",
+	Usage:       "View a dashboard from the provided configuration",
+	Action:      view,
+	Flags: []cli.Flag{
+
+		// Repo Flags
```
Comment (plyr4) [inline command/dashboard/view.go:24]:
```suggestion
		// Dashboard Flags
```

**Review by plyr4 — DISMISSED** [review]

**Review by plyr4 — APPROVED** [review]

File: action/dashboard/update.go [hunk 3]
```diff
@@ -0,0 +1,125 @@
...
+
+	// drop specified admins from the dashboard
+	if len(c.DropAdmins) > 0 {
+		newAdmins := []*api.User{}
+
+		for _, a := range dashboard.GetAdmins() {
+			if !slices.Contains(c.DropAdmins, a.GetName()) {
+				newAdmins = append(newAdmins, a)
+			}
+		}
```
Comment (wass3rw3rk) [inline action/dashboard/update.go:89]: do we need a check to make sure that there is at least one admin left and/or add server side?

**Review by wass3rw3rk — COMMENTED** [review]
looks good.. one question

File: action/dashboard/update.go (hunk 3 above)
Comment (ecrupper) [reply action/dashboard/update.go:89]: https://github.com/go-vela/server/blob/8a744e48d224197478bb48a468186783b04c44db/api/dashboard/create.go#L144-L167 This function uses the caller as a starting point for any `admins` input.

Comment (wass3r) [reply action/dashboard/update.go:89]: aha! i looked at update and didn't drill into that function. so, caller is always in it. someone amazing must have added that 😺

**Review by wass3r — APPROVED** [review]

**Pull request merged**